src/gsi/crtm_interface.f90

```diff
@@ -570,16 +581,29 @@ subroutine init_crtm(init_pass,mype_diaghdr,mype,nchanl,nreal,isis,obstype,radmo
 
  sensorlist(1)=isis
  quiet=.not. print_verbose
+
  if( crtm_coeffs_path /= "" ) then
     if(init_pass .and. mype==mype_diaghdr .and. print_verbose) &
         write(6,*)myname_,': crtm_init() on path "'//trim(crtm_coeffs_path)//'"'
     error_status = crtm_init(sensorlist,channelinfo,&
        Process_ID=mype,Output_Process_ID=mype_diaghdr, &
+! for crtm2.4.1 
```

Comment:
see the comment for Line 58 (the same for this kind of comments in the following)